shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDBBuilderTest.java

```diff
@@ -0,0 +1,212 @@
+package nf.fr.k49.shielddb.core;
+
+import nf.fr.k49.shielddb.core.shield.BottomShield;
+import nf.fr.k49.shielddb.core.shield.ShieldDBShield;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ShieldDBBuilderTest {
+
+    @Test
+    public void firstShieldPlacedAtBeginningOfShield() {
+        ShieldDBShield<String> anyFirstShield = new MockDBShield();
+        ShieldDBShield<String> anySecondShield = new MockDBShield();
+
+        List<String> build = ShieldDB.<String>builder()
+                .shield(anyFirstShield)
+                .shield(anySecondShield)
+                .build();
+
+        MatcherAssert.assertThat(build, Matchers.is(anyFirstShield));
```

Comment:
As you can see in the actual tests (https://github.com/kuroidoruido/ShieldDB/blob/master/shielddb-core/src/test/java/nf/fr/k49/shielddb/core/storage/FileStorageTest.java), the MatcherAssert methods are used through static import. I think it's more readable (avoid some noise when reading the tests). Can you change this?

Reply:
:)
For _me_, static imports are awful (don't ask why ) (but a little bit noise, really); but I know that many people love them.
I'll change ist the next days ;)